performance-test/locustfile.py

```diff
@@ -0,0 +1,136 @@
+import os
+import random
+import time
+import logging
+
+from locust import task, run_single_user, events
+from locust import FastHttpUser
+from locust.env import Environment
+from locust.exception import RescheduleTask
+
+from config_schema import Config, RequestTypeEnum
+
+config: Config
+
+
+@events.init.add_listener
+def on_locust_init(environment: Environment, **kwargs):
+    logging.info(f"on_locust_init called - loading config file")
+    with open("config.json") as configfile:
+        global config
+        config = Config.model_validate_json(configfile.read())
+
+
+class DataProviderUser(FastHttpUser):
+
+    host = 'https://publish.development.digital-land.info'
```

Comment:
is this host used anywhere?

Reply:
Yep - it's the default host which can be overridden when executing a test run.